src/module-1/card/stylesheets/card.css

```
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300&display=swap');
+*{
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+.card{
+  height: 486px;
+  max-width: 400px;
+  border-radius: 8px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.135216);
+
+  background: white;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.product{
+  height: 272px;
+  width: 237.19676208496094px;
```

```suggestion
  width: 238px;
```  
please define all `px`-values as integers, browsers do not use fractions

---

[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)